autosklearn/automl.py

```diff
@@ -1644,7 +1662,10 @@ def _get_runhistory_models_performance(self):
             endtime = pd.Timestamp(
                 time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(run_value.endtime))
             )
-            val_score = metric._optimum - (metric._sign * run_value.cost)
+            cost = run_value.cost
+            if not isinstance(self._metric, Scorer):
```

Comment:
When would that happen? self._metric is either a Scorer or a sequence of Scorers (whereas the latter is not possible due to line 1653)

Reply:
This will happen if we solve a multi-objective optimization problem. It can happen because we do not update `self._metric`. I will refactor to make `self._metric` always be a list and then check for `len(self._metrics) > 1`.